linux/switchtec_ioctl.h

```diff
@@ -34,9 +34,13 @@
 #define SWITCHTEC_IOCTL_PART_VENDOR7	12
 #define SWITCHTEC_IOCTL_PART_BL2_0	13
 #define SWITCHTEC_IOCTL_PART_BL2_1	14
+#define SWITCHTEC_IOCTL_PART_MAP_0	15
+#define SWITCHTEC_IOCTL_PART_MAP_1	16
+#define SWITCHTEC_IOCTL_PART_KEY_0	17
+#define SWITCHTEC_IOCTL_PART_KEY_1	18
```

Comment:
These should probably be added in the patch that adds the code in the ioctl.

Reply:
updated